.github/workflows/infra_drift_detection.yml

```diff
@@ -7,7 +7,7 @@ on:
         description: Environment where the resources will be deployed.
```

Comment:
I'd keep `prod` in this file as primary consumers are other repos

Reply:
I will disable the schedule? Because with prod the pipe will not work in this repo